api/rebac/v1/check.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package api.rebac.v1;
+import "google/api/annotations.proto";
+import "rebac/v1/relationships.proto";
+option go_package = "ciam-rebac/api/rebac/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.rebac.v1";
+
+service Check {
+	rpc Check (CheckRequest) returns (CheckResponse) {
+		option (google.api.http) = {
+			post: "/v1/check"
+			body: "*"
+		};
+	};
+}
+
+message CheckRequest {
+	Relationship relationships = 1;
```

Comment:
@akoserwal do you think it's a good idea to embed the whole Relationship type here? I'm worried it makes the API unnecessarily verbose. You'd have to instantiate both a CheckRequest and a Relationship. How SpiceDB and other implementations do it is nice, because it's easier to use (less objects to instantiate): https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.CheckPermission

Reply:
updated